IBPSA/BoundaryConditions/BESTEST.mo

```diff
@@ -0,0 +1,1718 @@
+<li>Relative humidity (&percnt;)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2) </li>
+</ul>
+<p><br>The following outputs shall be provided integrated hourly for the days specified for each test case in Table 3:</p>
+<ul>
+<li>Total incident horizontal solar radiation (Wh/m2)</li>
+<li>Total incident horizontal beam solar radiation (Wh/m2)</li>
+<li>Total incident horizontal diffuse solar radiation (Wh/m2) </li>
+</ul>
+<h4>Validation results</h4>
+<p>(Not available yet)</p>
+<h4>Implementation</h4>
+<p>To generate the data shown in this user guide, run </p>
+<p><span style=\"font-family: Courier New;\">cd IBPSA/Resources/src/BoundaryConditions/Validation/BESTEST</span></p>
+<p><span style=\"font-family: Courier New;\">python3 WeatherBESTEST.py</span></p>
+<h4>References</h4>
+<p>(Not available yet)</p>
+</html>"));
+  end UsersGuide;
+
+  package Validation
+    "Boundary conditions validation according to BESTEST specifications"
+    model WD100 "Test model for BESTEST weather data: base case"
+      extends Modelica.Icons.Example;
+      parameter Modelica.SIunits.Angle lat=0.6952170009469 "Latitude angle";
+      parameter Real rho=0 "Ground reflectance";
+
+      WeatherData.ReaderTMY3                          weaDat(
+        pAtmSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        ceiHeiSou=IBPSA.BoundaryConditions.Types.DataSource.Parameter,
+        ceiHei=1650,
+        totSkyCovSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        opaSkyCovSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        TDryBulSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        TDewPoiSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        TBlaSkySou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        relHumSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        winSpeSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        winDirSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        HInfHorSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        HSou=IBPSA.BoundaryConditions.Types.RadiationDataSource.File,
+        filNam=Modelica.Utilities.Files.loadResource(
+            "modelica://IBPSA/Resources/Data/BoundaryConditions/WeatherData/Validation/725650.mos"),
+        calTSky=IBPSA.BoundaryConditions.Types.SkyTemperatureCalculation.HorizontalRadiation)
+        annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+            rotation=90,
+            origin={0,-88})));
+
+      WeatherData.Bus weaBus annotation (Placement(transformation(extent={{-16,-82},
+                {14,-54}}), iconTransformation(extent={{-220,70},{-200,90}})));
+      ComRadIsoPerDir Azi000Til00(
+        til=IBPSA.Types.Tilt.Ceiling,
+        lat=lat,
+        azi=IBPSA.Types.Azimuth.S,
+        rho=rho) "Azimuth = Horizontal, Tilt = 0 °"
+        annotation (Placement(transformation(extent={{68,68},{88,88}})));
+      ComRadIsoPerDir Azi000Til90(
```

Comment:
Use lower case for instance names, e.g., `azi00....`

Reply:
Corrected all the radiation models names